src/diffusers/models/attention.py

```diff
@@ -202,6 +202,7 @@ def __init__(
         num_embeds_ada_norm: Optional[int] = None,
         attention_bias: bool = False,
         only_cross_attention: bool = False,
+        double_self_attention: bool = False,
```

Comment:
have we added this to the docstring so that the docs for it are auto-generated?

Reply:
Nope not adding yet - need to do it